netcore-ef-util/Extensions.cs

```diff
@@ -13,16 +14,27 @@ namespace SearchAThing.EFUtil
     public static partial class Util
     {
 
+        public static List<T> ExecSQL<T>(this DbContext context, string query, ILogger logger = null)
+        {
+            return ExecSQL<T>(context, query, logger);
+        }
+
+        public static List<T> ExecSQL<T>(this DbContext context, string query, params SqlParameter[] sqlParams)
+        {
+            return ExecSQL<T>(context, query, null, sqlParams);
+        }
```

Comment:
These additional signatures were necessary in order to allow both 
1. optional ILogger, and
2. varargs SqlParameter[]